eng/jacoco-test-coverage/pom.xml

```diff
@@ -410,47 +410,47 @@
     <dependency>
       <groupId>com.azure.spring</groupId>
       <artifactId>azure-identity-spring</artifactId>
-      <version>1.6.0-beta.1</version> <!-- {x-version-update;com.azure.spring:azure-identity-spring;current} -->
+      <version>1.6.0</version> <!-- {x-version-update;com.azure.spring:azure-identity-spring;current} -->
```

Comment:
Do we need to add azure-security-keyvault-jca and azure-spring-boot-starter-keyvault-certificates in this pom.xml?

Reply:
I am not sure, I will look into it.
It's not related with spring release.